server/tests/pom.xml

```diff
@@ -73,6 +73,18 @@
          <artifactId>h2</artifactId>
          <scope>test</scope>
       </dependency>
+      <dependency>
+         <groupId>mysql</groupId>
+         <artifactId>mysql-connector-java</artifactId>
+         <version>8.0.20</version>
+         <scope>test</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.postgresql</groupId>
+         <artifactId>postgresql</artifactId>
+         <version>42.2.5</version>
```

Comment:
We should define this version in the root pom.xml like we do for h2, e.g. <versionx.com.h2database.h2/>

Reply:
done